.asf.yaml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+github:
+  description: "Apache XTable (incubating) is a cross-table converter for lakehouse table formats that facilitates interoperability across data processing systems and query engines."
+  homepage: https://xtable.apache.org/
+  labels:
+    - apache-iceberg
+    - apache-hudi
+    - delta-lake
+  features:
+    wiki: false
+    issues: true
+    projects: true
+    discussions: true
+  enabled_merge_buttons:
+    # "squash and merge" replaces committer with noreply@github, and we don't want that
+    # See https://lists.apache.org/thread/vxxpt1x316kjryb4dptsbs95p66d9xrv
+    squash:  false
```

Comment:
When squash and merge is disabled, should all the PRs squash the changes before merging?
This is the recommendation in the discussion link above for Apache Drill.

Reply:
I think it can be done automatically via the rebase button. Otherwise it has to be done manually either by the PR author or by the maintainer who is going to merge the PR to main. Quite often I use the command line to squash and merge changes to the repo thus I am not using the GitHub UI much.